tests/unit/OrgPerson.spec.ts

```diff
@@ -16,238 +13,289 @@ const vuetify = new Vuetify({})
     expect((wrapper.vm as any).isDirector).toBe(true)
     expect((wrapper.vm as any).isCompletingParty).toBe(true)
+
     wrapper.destroy()
   })
 
   it('Loads the component and sets data for org', async () => {
-    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, -1, 0, null)
+    const wrapper = createComponent(validOrgData, NaN, 0, null)
     await Vue.nextTick()
+
     expect(wrapper.vm.$data.orgPerson).toStrictEqual(validOrgData)
     expect((wrapper.vm as any).isIncorporator).toBe(true)
     expect((wrapper.vm as any).isDirector).toBe(false)
     expect((wrapper.vm as any).isCompletingParty).toBe(false)
+
     wrapper.destroy()
   })
 
-  xit('Displays form data for org', async () => {
-    const wrapper: Wrapper<OrgPerson> = createComponent(validOrgData, -1, 0, null)
-    expect((<HTMLInputElement> wrapper.find(orgNameSelector).element).value)
-      .toEqual(validOrgData['officer']['orgName'])
+  it('Displays form data for person', async () => {
+    const wrapper = createComponent(validPersonData, 0, null, null)
     await Vue.nextTick()
+
+    expect((wrapper.find(firstNameSelector).element as HTMLInputElement).value)
+      .toEqual(validPersonData['officer']['firstName'])
+    expect((wrapper.find(middleNameSelector).element as HTMLInputElement).value)
+      .toEqual(validPersonData['officer']['middleName'])
+    expect((wrapper.find(lastNameSelector).element as HTMLInputElement).value)
+      .toEqual(validPersonData['officer']['lastName'])
+
     expect(wrapper.find(doneButtonSelector).attributes('disabled')).toBeUndefined()
-    expect(wrapper.find(removeButtonSelector).attributes('disabled')).toBeDefined()
+    expect(wrapper.find(removeButtonSelector).attributes('disabled')).toBeUndefined()
     expect(wrapper.find(cancelButtonSelector).attributes('disabled')).toBeUndefined()
+
     wrapper.destroy()
   })
 
-  it('Displays form data for person', async () => {
-    const wrapper: Wrapper<OrgPerson> = createComponent(validPersonData, 0, 0, null)
-    expect((<HTMLInputElement> wrapper.find(firstNameSelector).element).value)
-      .toEqual(validPersonData['officer']['firstName'])
-    expect((<HTMLInputElement> wrapper.find(middleNameSelector).element).value)
-      .toEqual(validPersonData['officer']['middleName'])
-    expect((<HTMLInputElement> wrapper.find(lastNameSelector).element).value)
-      .toEqual(validPersonData['officer']['lastName'])
+  it('Displays form data for org', async () => {
+    const wrapper = createComponent(validOrgData, NaN, 0, null)
     await Vue.nextTick()
+
+    expect((wrapper.find(orgNameSelector).element as HTMLInputElement).value)
+      .toEqual(validOrgData['officer']['orgName'])
+
     expect(wrapper.find(doneButtonSelector).attributes('disabled')).toBeUndefined()
-    expect(wrapper.find(removeButtonSelector).attributes('disabled')).toBeUndefined()
+    expect(wrapper.find(removeButtonSelector).attributes('disabled')).toBeDefined()
```

Comment:
This is correct: if we add an org (or person) then the Remove button is disabled.